### ISSUE TITLE
OutOfMemoryError

### ISSUE THREAD

**Issue by FranciscoBeccuti** [AmolGangadhare/flutter_barcode_scanner#31]
**Describe the bug**
When running the example on Android, when scanning the barcode, if you turn the flash on and off 10 times it gives an error.

**To Reproduce**
1. Go to ' Barcode scan'
2. Click on ' Start barcode scan'
'press the flash button 10 or more times'
3. See error

**Expected behavior**
The application does not end with OutOfMemoryError

**Smartphone (please complete the following information):**
 - Device: moto g / moto x4 (press 20 times )
 - OS: Android 7 / Android 9

**Additional context**
Output of error.
```
E/AndroidRuntime(11925): FATAL EXCEPTION: main
E/AndroidRuntime(11925): Process: com.amolg.flutterbarcodescannerexample, PID: 11925
E/AndroidRuntime(11925): java.lang.OutOfMemoryError: Failed to allocate a 1843213 byte allocation with 910544 free bytes and 885KB until OOM
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.camera.CameraSource.createPreviewBuffer(CameraSource.java:931)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.camera.CameraSource.createCamera(CameraSource.java:751)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.camera.CameraSource.start(CameraSource.java:323)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.camera.CameraSourcePreview.startIfReady(CameraSourcePreview.java:92)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.camera.CameraSourcePreview.start(CameraSourcePreview.java:66)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.camera.CameraSourcePreview.start(CameraSourcePreview.java:73)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.BarcodeCaptureActivity.startCameraSource(BarcodeCaptureActivity.java:325)
E/AndroidRuntime(11925): 	at com.amolg.flutterbarcodescanner.BarcodeCaptureActivity.onClick(BarcodeCaptureActivity.java:392)
E/AndroidRuntime(11925): 	at android.view.View.performClick(View.java:5637)
E/AndroidRuntime(11925): 	at android.view.View$PerformClick.run(View.java:22433)
E/AndroidRuntime(11925): 	at android.os.Handler.handleCallback(Handler.java:751)
E/AndroidRuntime(11925): 	at android.os.Handler.dispatchMessage(Handler.java:95)
E/AndroidRuntime(11925): 	at android.os.Looper.loop(Looper.java:154)
E/AndroidRuntime(11925): 	at android.app.ActivityThread.main(ActivityThread.java:6128)
E/AndroidRuntime(11925): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(11925): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
E/AndroidRuntime(11925): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

**Comment by AmolGangadhare** [comment]
@FranciscoBeccuti 
Resolved in V 0.1.6 release.

If possible can you please acknowledge that it is resolved?

**Comment by FranciscoBeccuti** [comment]
Hi.
It works correctly and even better!
Thank you!
